tests/stress/stress_test.go

```diff
@@ -13,19 +13,21 @@ type StressTest struct {
 	Layer0    *clients.Layer0TestClient
```

Comment:
Now that I look at it, the `type StressTest struct` seems heavy-handed to me (same problem exists in system tests). 
I think it would be easier to just have `func NewStressTest` return the `APIClient` and `Terraform` objects.

```
terraform, apiClient := NewStressTest(b, "cases/modules", tfvars)
terraform.Apply()
defer terraform.Destroy()

b.Run("ListEnvironments", func(b *testing.B) {
    for n := 0; n < b.N; n++ {
        apiClient.ListEnvironments()
    }
})
```

Reply:
✅